cinema/models.py

```diff
@@ -39,7 +39,7 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
+    genres = models.ManyToManyField(Genre, related_name="movies")
     actors = models.ManyToManyField(Actor)
```

Comment:
why did you only add `related_name` here? You could also add it for `actors` field